extensions/packageManager/src/client/Library.tsx

```diff
@@ -294,16 +295,20 @@ const Library: React.FC = () => {
   };
 
   const getInstalledLibraries = async () => {
-    try {
-      updateInstalledComponents([]);
-      const response = await getInstalledComponentsAPI(currentProjectId);
-      updateInstalledComponents(response.data.components);
-    } catch (err) {
-      setApplicationLevelError({
-        status: err.response.status,
-        message: err.response && err.response.data.message ? err.response.data.message : err,
-        summary: strings.libraryError,
-      });
+    if (!isLoadingInstalled) {
+      setIsLoadingInstalled(true);
+      try {
+        updateInstalledComponents([]);
```

Comment:
why does this function invocation need to accept an empty array?

Reply:
That is resetting the state to an empty list while it loads...